index.html

```diff
@@ -0,0 +1,276 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    
+    <link rel="stylesheet" href="style.css">
+
+    <title>Sanam's Portfolio</title>
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap" rel="stylesheet">
+
+</head>
+<body>
+    
+<section class="head">
+      <header class="header">
+        <nav class="nav-menu">
+            <a href="#"> <img src="images\Logo Placeholder.svg" alt="logo"></a>
+
+            <div class="nav-menu">
+                <ul class="menu-desktop" href="#">
+                    <li><a href="my-work">Portfolio</a></li>
+                    <li><a href="about-me">About</a></li>
+                    <li><a href="my-work">Contact</a></li>
+                </ul>
+                <img src="images\Union.svg" alt="menu-icon" class="menu hamburger-menu"/>
+            </div>
+
+            <div class="menu-hide mobile-menu-container">
+                <div class="mobile-menu">
+                  <div class="close-menu">
+                    <img class="close-menu-icon" src="images\Icon.svg" alt="close-icon">
+                  </div>
+                  <ul class="mobile-menu-item-list">
+                    <li class="mobile-menu-item"><a href="#my-works">Portfolio</a></li>
+                    <li class="mobile-menu-item"><a href="#about-me">About</a></li>
+                    <li class="mobile-menu-item"><a href="#contact-me">Contact</a></li>
+                  </ul>
+                  </div>
+                  </div>
+        </nav>
+      </header>
+
+      <section class="headline-section blur-bg">
+        <div class="h-container h-content">
+          <div class="h-main">
+            <h1>I'm Sanam,<br />Glad to see you</h1>
+            <p>I'm a software developer! I can help you build a product , feature or website Look through <br> some of my work and experience! If you like what you see and have a project you need <br> coded, don't hestiate to contact me.</p>
+            <a href="#" class="connect">LET'S CONNECT</a>

```

Comment:
Kindly make use of the color like what we have in the template.
Also, it will be great if you can implement the Active, Hover, and Pressed states, like the Figma design.
<img width="751" alt="image" src="https://user-images.githubusercontent.com/73854999/227648204-b7f1bec8-4749-4cb5-93d1-122d28951cb5.png">